src/sidebar.ts

```diff
@@ -64,103 +64,132 @@ const openIdSSO: SidebarItemConfig = {
   ],
 }
 
-const referenceSidebar: ExtendSidebar = [
+const api: ExtendSidebar = [
   homeLink,
+  "reference/api",
+
+  "ecosystem/api-design",
   {
     type: "category",
-    label: "Reference",
+    label: "Operations",
     collapsed: false,
     collapsible: false,
+    link: {
+      type: "generated-index",
+      slug: "/category/operations-reference",
+    },
     items: [
-      "reference/api",
-      "sdk",
-      "ecosystem/api-design",
-      {
-        type: "category",
-        label: "Operations reference",
-        collapsed: false,
-        collapsible: false,
-        link: {
-          type: "generated-index",
-        },
-        items: [
-          "concepts/personal-access-token",
-          "guides/cors",
-          "guides/custom-domains",
-          "guides/api-rest-pagination",
-          "guides/workspaces",
-          "guides/rate-limits",
-          "guides/ip-allowlist",
-          "api/eventual-consistency",
-          "console/usage-billing",
-        ],
-      },
+      "concepts/personal-access-token",
+      "guides/cors",
+      "guides/custom-domains",
+      "guides/api-rest-pagination",
+      "guides/workspaces",
+      "guides/rate-limits",
+      "guides/ip-allowlist",
+      "api/eventual-consistency",
+      "console/usage-billing",
```

Comment:
things like rate limits and ip allow lists are things directly related to APIs, I agree, but usage billing, workspaces, custom domains they have docs that also describe the console part of it, and it doesn't fit the current Reference->Rest API -> Operations in my view, wdyt?

Reply:
at the same time RL IP WL etc also perfectly fit Managge -> Platform

Reply:
Thanks @piotrmsc 

I completely agree. Let me think about this and see where/how to fit them better.